src/uwloc/data/main.py

```diff
@@ -18,34 +20,52 @@ def fail(msg: str) -> None:
 @app.command()
 def initdb(
     dbpath: str,
+    wavs_dir: str,
     max_units: Annotated[
         int, typer.Argument(help="Maximum number of units (rows) to store in the array")
     ] = MAX_UNITS,
     max_hrs: Annotated[int, typer.Argument(help="Maximum amount of hours to store in the array")] = MAX_HOURS,
 ) -> None:
-    typer.echo(f"Initializing TileDB in {dbpath}")
-    create(dbpath, max_units, max_hrs)
+    start_date: datetime = _find_start_date(wavs_dir)
+    typer.echo(f"Initializing TileDB in {dbpath} with start date: {start_date}")
+    create(dbpath, start_date, max_units, max_hrs)
+    import_wavs(dbpath, wavs_dir)
 
 
 @app.command(name="import")
-def import_wavs(wavs_dir: str, dbpath: str) -> None:
+def import_wavs(
+    dbpath: str,
+    wavs_dir: str,
+) -> None:
     typer.echo(f"Uploading WAV files from {wavs_dir} {dbpath}")
     if not os.path.isdir(dbpath):
         fail(f"TileDB '{dbpath}' not found. Run initdb command first.")
 
-    files = glob.iglob(os.path.join(wavs_dir, "**/*.[wW][aA][vV]"), recursive=True)
-    # TODO: Handle multiple recording files from the same device. These need to be sorted
-    # and laid out correctly in the array
+    files = sorted(glob.iglob(os.path.join(wavs_dir, WAV_PATTERN), recursive=True))
+    for f in files:
+        try:
+            (device, timestamp, rate, data) = read_wav(f)
```

Comment:
```suggestion
            device, timestamp, rate, data = read_wav(f)
```
No need for extra parentheses. Python de-structuring would make simply listing out the variables be cleaner to read.